src/app/icd/ICDClientInfoPersistentStorage.h

```diff
@@ -0,0 +1,178 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      This file defines the interface to store ICD Client Persistent information.
+ */
+
+#pragma once
+
+#include <crypto/CHIPCryptoPAL.h>
+#include <crypto/SessionKeystore.h>
+#include <lib/core/CHIPConfig.h>
+#include <lib/core/CHIPPersistentStorageDelegate.h>
+#include <lib/core/DataModelTypes.h>
+#include <lib/support/CodeUtils.h>
+#include <lib/support/CommonIterator.h>
+#include <lib/support/StorageKeyName.h>
+#include <stddef.h>
+
+namespace chip {
+namespace Crypto {
+using SymmetricKeystore = SessionKeystore;
+}
+} // namespace chip
+
+namespace chip {
+namespace app {
+
+using SymmetricKeystore = Crypto::SessionKeystore;
```

Comment:
you are polluting the global namespace. Please remove this.

Reply:
removed, thanks